database/schema.sql

```diff
@@ -202,9 +202,9 @@ CREATE TABLE rotation_move_history (
   new_bucket INTEGER NOT NULL,
   PRIMARY KEY (event_id),
   UNIQUE (prefrosh_id, old_bucket, new_bucket),
-  FOREIGN KEY (prefrosh_id) REFERENCES rotation_prefrosh (prefrosh_id),
-  FOREIGN KEY (old_bucket) REFERENCES rotation_buckets (bucket_id),
-  FOREIGN KEY (new_bucket) REFERENCES rotation_buckets (bucket_id)
+  FOREIGN KEY (prefrosh_id) REFERENCES rotation_prefrosh (prefrosh_id)
+  -- FOREIGN KEY (old_bucket) REFERENCES rotation_buckets (bucket_id),
+  -- FOREIGN KEY (new_bucket) REFERENCES rotation_buckets (bucket_id)
```

Comment:
How come you're removing these foreign keys?

Reply:
They raised errors when I tried to run this file. You can uncomment them if you can get them to work, but using the Ruddock server's version of MySQL, I was not able to.

Reply:
You've got a missing comma at the end of the prefrosh_id foreign key.

Reply:
Come to think of it, you probably deleted it intentionally. For some reason I'm not getting the errors you are. :\

Reply:
Works for me. I'm going to revert these changes.